### ISSUE TITLE
Cannot email to more than one email

### ISSUE THREAD

**Issue by bogdanez** [ActiveForums/ActiveForums#400]
Cannot email the topic to more than one email, even if I separate two email with a comma or a semicolon, or even a space.

What prevents this and how to fix/work around it?

I need to send the same message to up to 15 people on the committee, and it's tedious, at least.

Any help is welcome! Thanks in advance!

**Comment by artfuld** [comment]
Hello @bogdanez do you mean from this page 
![image](https://user-images.githubusercontent.com/6116748/38212649-cf3731ac-36be-11e8-9a30-34ef07ff1354.png)
That page seems really clear about only sending to one person. I will label this as a feature and if someone have the time they could fix a similar page for sending to several addresses.
Do you send to members of the site only?

**Comment by bogdanez** [comment]
@artfuld, yes, this is the page in question. We do want to send to all member of the committee, so yes, they are all members of the site. I was hoping we could hook into the DNN roles or something...

**Comment by artfuld** [comment]
That would be a nice feature. 
Have you checked the auto subscription feature? You could set up the role for the committee members to always subscripe to threads created in a specific forum (or group)

**Comment by bogdanez** [comment]
Yes, this is already part of the plan. We will have committee members automatically subscribed to all new topics. However, I'm not sure if they can unsubscribe manually, and if they can, then there is no good way to notify all of them of something mandatory, or if the reminder link is to the topic that has already been posted and hence "not new".

It seems that a good "manual" method would be very useful.
